joomla/joomla-cms
# [5.2] Rename update SQL scripts 5.2.0-2024-09-24.sql to 5.2.1-2024-09-24.sql

Pull Request for Issue # .

### Summary of Changes

The update SQL scripts "5.2.0-2024-09-24.sql" which were added with PR #44133 should have been renamed to "5.2.1-2024-09-24.sql" before merging because 5.2.0 was already released at that time.

This PR here does that now.

If a 5.2.1 will be released in the next days which does not contain these files because it only fixes the file system permissions, they have to be renamed again to "5.2.2-2024-09-24.sql".

The wrong file name is not a problem for updating as the date is newer than the newest 5.2.0 script present before an update, so the version of the file is still newer than the schema version in the database before the update, and so the script will run.

But it should have the right version in its name so nobody is confused.

### Testing Instructions

Code review, and check if the files "5.2.0-2024-09-24.sql" can be found in a 5.2.0 installation or update package.

### Actual result BEFORE applying this Pull Request

Update SQL scripts are named "5.2.0-2024-09-24.sql" but can't be found in a 5.2.0.

### Expected result AFTER applying this Pull Request

Update SQL scripts are named "5.2.1-2024-09-24.sql".

### Link to documentations
Please select:
